library/styleguideComponents/StyleGuide/index.js

```diff
@@ -74,6 +74,11 @@ class StyleGuide extends React.Component {
     render() {
         const { children, toc } = this.props;
         const components = toc.props.sections[1].components;
+        const styleTwitterIcon = {
+            width: '18px',
+            height: '15px',
+            color: '#ffffff',
+        };
```

Comment:
you can move these styles outside the component because here a new object will be created in each rerender